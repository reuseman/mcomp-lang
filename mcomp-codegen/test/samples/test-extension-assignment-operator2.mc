component EntryPoint provides App { 
  def main() : int {
    var i : float;

    i = 0.0;
    print(i);
    
    i += 3.0;
    print(i);

    i *= 3.0;
    print(i);

    i -= 2.0;
    print(i);

    i /= 2.0;
    print(i);

    return 0;
  }
}