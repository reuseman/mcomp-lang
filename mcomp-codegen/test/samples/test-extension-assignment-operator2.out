0.00
3.00
9.00
7.00
3.50
